R/robustbayesianmetaanalysis.R

```diff
@@ -1460,16 +1464,16 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
     title    <- gettext("Effect size vs Heterogeneity")
     position <- 4
   }
-  title <- gettextf("%1$s (%2$s)", title, ifelse(options[["plots_type"]] == "conditional", gettext("Conditional"), gettext("Model Averaged")))
+  title <- gettextf("%1$s (%2$s)", title, ifelse(options[["plotsType"]] == "conditional", gettext("Conditional"), gettext("Model Averaged")))
   
   if (all(parameters %in% "theta")) {
-    height <- 250 + ncol(fit[["RoBMA"]][["samples"]][["averaged"]][["theta"]]) * if (options[["plots_theta_show"]] == "both") 55 else  20
+    height <- 250 + ncol(fit[["RoBMA"]][["samples"]][["averaged"]][["theta"]]) * if (options[["plotsThetaShow"]] == "both") 55 else  20
```

Comment:
Are there specific reasons for the numbers 250, 55, 20 here? And do they appear multiple times? I just wonder how this will interact with plot editing

Reply:
they make the default figures look good, just a lot of manual tunning

Reply:
Perhaps have these numbers defined somewhere centrally, so in case they need changing then you can change them easily.

Reply:
hmm, as far as I have seen in other analyses, the plot dimensions were always defined at the plot definitions